Library/Formula/relx.rb

```diff
@@ -0,0 +1,43 @@
+class Relx < Formula
+  homepage "https://github.com/erlware/relx"
+  url "https://github.com/erlware/relx.git", :tag => "v1.2.0"
+
+  head "https://github.com/erlware/relx.git"
+
+  depends_on "erlang"
+  depends_on "rebar"
+
+  def install
+    system "make", "escript"
+    bin.install "relx"
```

Comment:
If there's no submodules let's use https://github.com/erlware/relx/archive/v1.2.0.tar.gz as the URL